lib/src/default_emoji_set.dart

```diff
@@ -0,0 +1,1624 @@
+    Emoji('😑', 'Expressionless Face'),
+    Emoji('😶', 'Face Without Mouth'),
+    Emoji('😏', 'Smirking Face'),
+    Emoji('😒', 'Unamused Face'),
+    Emoji('🙄', 'Face With Rolling Eyes'),
+    Emoji('😬', 'Grimacing Face'),
+    Emoji('🤥', 'Lying Face'),
+    Emoji('😌', 'Relieved Face'),
+    Emoji('😔', 'Pensive Face'),
+    Emoji('😪', 'Sleepy Face'),
+    Emoji('🤤', 'Drooling Face'),
+    Emoji('😴', 'Sleeping Face'),
+    Emoji('😷', 'Face With Medical Mask'),
+    Emoji('🤒', 'Face With Thermometer'),
+    Emoji('🤕', 'Face With Head-Bandage'),
+    Emoji('🤢', 'Nauseated Face'),
+    Emoji('🤮', 'Face Vomiting'),
+    Emoji('🤧', 'Sneezing Face'),
+    Emoji('🥵', 'Hot Face'),
+    Emoji('🥶', 'Cold Face'),
+    Emoji('🥴', 'Woozy Face'),
+    Emoji('😵', 'Dizzy Face'),
+    Emoji('🤯', 'Exploding Head'),
+    Emoji('🤠', 'Cowboy Hat Face'),
+    Emoji('🥳', 'Partying Face'),
+    Emoji('😎', 'Smiling Face With Sunglasses'),
+    Emoji('🤓', 'Nerd Face'),
+    Emoji('🧐', 'Face With Monocle'),
+    Emoji('😕', 'Confused Face'),
+    Emoji('😟', 'Worried Face'),
+    Emoji('🙁', 'Slightly Frowning Face'),
+    Emoji('☹', 'Frowning Face'),
+    Emoji('😮', 'Face With Open Mouth'),
+    Emoji('😯', 'Hushed Face'),
+    Emoji('😲', 'Astonished Face'),
+    Emoji('😳', 'Flushed Face'),
+    Emoji('🥺', 'Pleading Face'),
+    Emoji('😦', 'Frowning Face With Open Mouth'),
+    Emoji('😧', 'Anguished Face'),
+    Emoji('😨', 'Fearful Face'),
+    Emoji('😰', 'Anxious Face With Sweat'),
+    Emoji('😥', 'Sad but Relieved Face'),
+    Emoji('😢', 'Crying Face'),
+    Emoji('😭', 'Loudly Crying Face'),
+    Emoji('😱', 'Face Screaming in Fear'),
+    Emoji('😖', 'Confounded Face'),
+    Emoji('😣', 'Persevering Face'),
+    Emoji('😞', 'Disappointed Face'),
+    Emoji('😓', 'Downcast Face With Sweat'),
+    Emoji('😩', 'Weary Face'),
+    Emoji('😫', 'Tired Face'),
+    Emoji('😤', 'Face With Steam From Nose'),
+    Emoji('😡', 'Pouting Face'),
+    Emoji('😠', 'Angry Face'),
+    Emoji('🤬', 'Face With Symbols on Mouth'),
+    Emoji('😈', 'Smiling Face With Horns'),
+    Emoji('👿', 'Angry Face With Horns'),
+    Emoji('💀', 'Skull'),
+    Emoji('☠', 'Skull and Crossbones'),
+    Emoji('💩', 'Pile of Poo'),
+    Emoji('🤡', 'Clown Face'),
+    Emoji('👹', 'Ogre'),
+    Emoji('👺', 'Goblin'),
+    Emoji('👻', 'Ghost'),
+    Emoji('👽', 'Alien'),
+    Emoji('👾', 'Alien Monster'),
+    Emoji('🤖', 'Robot Face'),
+    Emoji('😺', 'Grinning Cat Face'),
+    Emoji('😸', 'Grinning Cat Face With Smiling Eyes'),
+    Emoji('😹', 'Cat Face With Tears of Joy'),
+    Emoji('😻', 'Smiling Cat Face With Heart-Eyes'),
+    Emoji('😼', 'Cat Face With Wry Smile'),
+    Emoji('😽', 'Kissing Cat Face'),
+    Emoji('🙀', 'Weary Cat Face'),
+    Emoji('😿', 'Crying Cat Face'),
+    Emoji('😾', 'Pouting Cat Face'),
+    Emoji('💋', 'Kiss Mark'),
+    Emoji('👋', 'Waving Hand', hasSkinTone: true),
+    Emoji('🤚', 'Raised Back of Hand', hasSkinTone: true),
+    Emoji('🖐', 'Hand With Fingers Splayed', hasSkinTone: true),
```

Comment:
That's actually awesome! Makes it so much easier to maintain ! 💯 👏

Reply:
Glad you like it.